apps/payments/src/pages/payments/[requestId].tsx

```diff
@@ -50,7 +74,21 @@ export default function PaymentDetails() {
 
   return (
     <>
-      <Stack my={3} spacing={2}>
+      <Stack spacing={2}>
+        {data.descrizioneTipoPratica && (
+          <SectionTitle label={data.descrizioneTipoPratica} />
+        )}
+        <Box width="max-content">
+          <Button
```

Comment:
let's check if the button should be displayed only under certain conditions _(example: particular practice status, even if we know that the practice status is not an enumeration...)_